packages/state-transition/src/util/weakSubjectivity.ts

```diff
@@ -97,6 +129,18 @@ export function computeWeakSubjectivityPeriodFromConstituents(
   return wsPeriod;
 }
 
+export function computeWeakSubjectivityPeriodFromConstituentsElectra(
+  totalBalanceByIncrement: number,
+  balanceChurnLimit: number, // Note this is not the same as churnLimit in `computeWeakSubjectivityPeriodFromConstituentsPhase0`
```

Comment:
the function signature is really not ideal, it would be better to get it closer to how the spec works but I can see why we do it like this as well